packages/admin-frontend/pages/featured.js

```diff
@@ -251,26 +287,73 @@ export default class EditFeaturedContent extends React.Component<Props, State> {
                     </>
                   )}
                 />
-                <Field
-                  name="imageUrl"
-                  render={({ input, meta }) => (
-                    <>
-                      <TextField
-                        margin="normal"
-                        fullWidth
-                        error={meta.error && meta.touched}
-                        type="url"
-                        disabled={selectedLanguage === ''}
-                        label="Image Url"
-                        {...input}
-                      />
-                      {meta.error &&
-                        meta.touched && (
-                          <FormHelperText error>{meta.error}</FormHelperText>
-                        )}
-                    </>
-                  )}
-                />
+
+                <div
+                  css={{
+                    display: 'flex',
+                    flexWrap: 'wrap',
+                    alignItems: 'center'
+                  }}
+                >
+                  <div css={{ flex: 1 }}>
+                    <Field
+                      name="imageUrl"
+                      render={({ input, meta }) => (
+                        <>
+                          <TextField
+                            fullWidth
+                            margin="normal"
+                            error={meta.error && meta.touched}
+                            type="url"
+                            disabled={selectedLanguage === ''}
+                            label="Image Url"
+                            {...input}
+                          />
+                          {meta.error &&
+                            meta.touched && (
+                              <FormHelperText error>
+                                {meta.error}
+                              </FormHelperText>
+                            )}
+                        </>
+                      )}
+                    />
+                  </div>
+
+                  <div css={{ margin: '20px' }}>or</div>
+
+                  <div css={{ align: 'center' }}>
+                    <Field
+                      name="selectedFile"
+                      render={({ input }) => (
```

Comment:
I don't think the file should be part of the form state? Looks like it isn't tied to the form (but a separate upload step). But the file is still in the page state and the form state?

Reply:
Det var slik jeg tenkte før jeg startet å implementere dette, men for at man skal kunne trykke "discard changes" og resette form ved endringer så må jeg inkludere valgt fil i formet. På onSubmit så blir da denne verdien slettet slikt at det ikke blir med i POST/PUT.